.babelrc.js

```diff
@@ -20,4 +20,12 @@ module.exports = {
             },
         ],
     ],
+    env: {
+        production: {
+            plugins: [['emotion', { sourceMap: false }]],
+        },
+        development: {
+            plugins: [['emotion', { sourceMap: true }]],
+        },
```

Comment:
Could we set `sourceMap` to `true` always? Or does this add extra weight?

Reply:
SourceMaps in production aren't recommended, as stated in Emotion's [docs](https://github.com/emotion-js/emotion/blob/master/docs/source-maps.md) state:

> We do not advise using sourceMaps in production. The source maps can add significant size to your bundle.